std_msgs/String[] new_names